onisite/settings_base.py

```diff
@@ -2,24 +2,25 @@
 # DJANGO 1.9 DEFAULT SETTINGS
 ####################################################################
 """
-Django settings for onisite project.
+Django settings for openoni project.
```

Comment:
I believe this was probably `onisite` because the "app" as far as django is concerned is called `onisite`?

Reply:
Correct. I just figured this would be more clear to those less fluent in Django :man_shrugging: